### PR TITLE
Implement sign-up page

### PR DESCRIPTION
**Pull request by valmdz**
# Description

Implement the Sign up page using:
1. Side banner #140 
2. Sign up form #131 
 
Fixes # (issue)

# How to test?

Go to:
http://localhost:3000/sign-up

Screenshot (2513)
Screenshot (2514)
Screenshot (2515)

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/SignUp/SignUp.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+nav {
+  display: none;
+}
+
+div.footer {
+  display: none;
+}
+
```
Comment (bytewiz) [inline src/client/containers/SignUp/SignUp.css:22]: No global styles, scope them with a class name

**Review by bytewiz — CHANGES_REQUESTED** [review]

File: src/client/containers/SignUp/SignUp.css (hunk 1 above)
Comment (valmdz) [reply src/client/containers/SignUp/SignUp.css:34]: @bytewiz would you suggest that I give the nav and the footer classes on App.js? I know it is not very good to use the global styles, but as the css is just being imported for the index of the Sign-up page, can it have a negative impact somewhere else?

File: src/client/containers/SignUp/index.js [hunk 2]
```diff
@@ -53,5 +56,22 @@ export default function SignUpContainer() {
...
   if (isLoading) return <Loader />;
-  return <SignUp onSubmit={onSubmit} />;
+  return (
+    <div>
+      <div className="split sign-up-left">
+        <div>
+          <SideBanner
+            image={signUpImg}
+            style={{ backgroundColor: '#F1C644' }}
+            className="side-banner-signUp"
```
Comment (kamalrsa) [inline src/client/containers/SignUp/index.js:66]: I couldont find the class "side-banner-signup" in signUp.css

Comment (valmdz) [reply src/client/containers/SignUp/index.js]: oh good catch, I will remove it

**Comment by dpfernandes** [comment]
👍

**Pull request merged**